redash/templates/forgot.html

```diff
@@ -13,7 +13,7 @@
 <form role="form" method="post">
     <div class="form-group ">
         <label for="email">Enter the email address you used for this account:</label>
-        <input type="text" class="form-control" name="email" value="{{email}}">
+        <input type="email" class="form-control" name="email" value="{{email}}">
```

Comment:
Here we can keep the email type, as it makes no sense to use the "forgot password" page for non emails.

My suggestion: update the pull request to change the type only here.